pkg/asset/installconfig/powervs/session.go

```diff
@@ -21,8 +25,9 @@ var (
 
 // Session is an object representing a session for the IBM Power VS API.
 type Session struct {
-	Session *ibmpisession.IBMPISession
-	Creds   *UserCredentials
+	Session    *ibmpisession.IBMPISession
+	Creds      *UserCredentials
+	OsOverride string
```

Comment:
this override var doesn't really fit into the concept of a session. it's a purely openshift installer concept